lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -586,4 +601,33 @@ defmodule RealtimeWeb.RealtimeChannel do
     Logger.error("Start channel error: " <> error_msg)
     {:error, %{reason: error_msg}}
   end
+
+  defp assign_policies(false, db_conn, sub_topic, access_token, claims, socket) do
+    channel =
+      case ChannelsCache.get_channel_by_name(sub_topic, db_conn) do
```

Comment:
Thinking ... this cache needs to be keyed per Channel / client JWT because the response depends on the JWT passed in the query. 

And it needs to get invalidated when the JWT expires. Which actually you can set on individual keys with Cachex.

Reply:
can this be tackled at a different PR? maybe even remove cache for now and tackle cache in a different PR